tests/cmdline/commands/test_calcjob.py

```diff
@@ -356,3 +356,42 @@ def test_calcjob_remotecat(self):
         options = [str(self.result_job.uuid), 'fileA.txt']
         result = self.cli_runner.invoke(command.calcjob_remotecat, options)
         assert result.stdout == 'test stringA'
+
+    def test_calcjob_gotocomputer(self, monkeypatch):
+        """Test verdi calcjob gotocomputer"""
+
+        from unittest.mock import patch
```

Comment:
why use `unittest.mock` when there is `monkeypatch`? Fine to keep the former if you prefer (even though we typically use the `monkeypatch` fixture) but if you do, get rid of the `monkeypatch` argument

Reply:
yes, you're right, I deleted `monkeypatch`.
Here I preferred `patch` because I could use it as context manager.